### ISSUE TITLE
Bundles stuck in uploading results can't be killed

### ISSUE THREAD

**Issue by EntilZha** [codalab/codalab-worksheets#3171]
**Describe the bug**
It seems like my runtime quota is being consumed by an unkillable job. I originally started the job on December 10, but it seems to have failed uploading its results yet still stuck in the running state. I attempted to kill it a few times with no success.

Ordinarily, this wouldn't be an issue, but it seems like I can't submit jobs even if I use my own queue/worker due to getting out of quota errors.

**Codalab Information**
- *Your username*: entilzha
- *UUID of worksheet where problem occurred*: https://worksheets.codalab.org/bundles/0xa8a4932a37b74214a0828b6f5da36804
- *UUID of problem bundle (please don't delete)*:
- *Error message received from Codalab (if applicable)*:
 - *Browser (if on Web)*; chrome

**To Reproduce**
Steps to reproduce the behavior :
1. Go to bundle view
2. See running state, see history of actions which has `kill kill kill kill`
3. See error
4. Attempt to kill the job with the web UI in the dashboard of running jobs, does not kill.

**Expected behavior**
The job should have been killed and stopped counting against quota

**Comment by epicfaace** [comment]
This is probably happening due to #3149.

**Comment by epicfaace** [comment]
@EntilZha Sorry you're facing this issue! While we're looking into it, we've increased your time quota so you could at least be unblocked and should be able to submit further jobs.

**Comment by EntilZha** [comment]
Thanks! I'm not certain how the scheduling logic work, but perhaps there is a bug somewhere that being out of quota is preventing jobs from running on local workers. I haven't debugged it thoroughly yet, so didn't want to open an issue until I confirmed it a bit more though.

**Comment by EntilZha** [comment]
I don't have the same error, but this bundle also seems stuck. I'm not certain, but I think its stuck since the worker running the job crashed (my desktop has some PSU issues atm). `cl kill 0x880204db8c5f4c30a1d2c37a5d3740d5` doesn't seem to be doing the trick.
https://worksheets.codalab.org/bundles/0x880204db8c5f4c30a1d2c37a5d3740d5

In this case, it seems to be causing my worker queue to be blocked (since a job is running); the workaround I think would be to make a new queue/tag, although slightly annoying since I'd have to mimic/rerun each job I already queued up.

**Comment by epicfaace** [comment]
This refers to the same error ("Invalid archive upload") that is referred to in #2722; however, this issue brings up a separate issue, that being that sometimes bundles can't be killed if they're in the middle of uploading results. I'll rename this issue so that it only concerns the latter issue.

**Comment by epicfaace** [comment]
Closing this in favor of #3233.
